pycaching/geocaching.py

```diff
@@ -363,19 +364,17 @@ def search_quick(self, area, *, strict=False, zoom=None):
 
     # add some shortcuts ------------------------------------------------------
 
-    def search_rect(self, rect: Rectangle, *, per_query: int = 50, sortby: str = 'datelastvisited', origin=None):
+    def search_rect(self, rect: Rectangle, *, per_query: int = 50, sort_by: str = 'datelastvisited', origin=None):
```

Comment:
Why is the default parameter of `sort_by` not using the enumeration entry? Anyway, the specified type does not seem to be correct, as with the current implementation this could either be a string or an enum entry - or am I missing something?

Reply:
I suppose the goal here was to make the interface simpler for end-user. I am also for using enum type as a default, but we can allow both here by leveraging the idempotence of Enums:
```python
>>> class Foo(enum.Enum):
...   a = 1
... 
>>> Foo(1)
<Foo.a: 1>
>>> Foo(Foo.a)
<Foo.a: 1>
```

Reply:
:+1: fixed